lib/graphql/schema/printer.rb

```diff
@@ -132,16 +132,22 @@ def print_input_value(arg)
           def print_value(value, type)
             case type
             when FLOAT_TYPE
+              return 'null' if value.nil?
```

Comment:
can we put it above the case? 

```ruby 
if value.nil? 
  "null" 
else 
  case ... 
end 
```

? 

(I see that NonNullType doesn't have this early return, but it _does_ make a recursive call, so if value is `nil`, then it will get to that condition _eventually_!)